src/web/components/App.tsx

```diff
@@ -364,6 +365,20 @@ export const App = ({ CAPI, NAV }: Props) => {
                     />
                 </HydrateOnce>
             ))}
+            {CAPI.quizAtoms.map((quizAtoms, index) => (
+                <HydrateOnce
+                    key={index}
+                    root="quiz-atom"
+                    index={quizAtoms.quizIndex}
+                >
+                    <QuizBlockComponent
+                        id={quizAtoms.id}
+                        questions={quizAtoms.questions}
+                        resultBuckets={quizAtoms.resultBuckets}
+                        quizType={quizAtoms.quizType}
+                    />
```

Comment:
If we remove spacing from `QuizBlockComponent` then maybe we don't need it at all? Without the css it is just a thing that makes a choice about which atom to use. Maybe we can inline that here? This has the added bonus of maintaining a similar pattern to the other atoms and not having to use the *Component conovention

Reply:
I originally did that, but I was unsure how to wrapp CSS to resolve padding bottom issue. Originally I thought of putting the CSS in `src/web/lib/ArticleRenderer.tsx` but felt this wasnt the right place for CSS. Adding CSS in `App.tsx` also doesnt feel right either and I suspected that we would of had some issues with hydration (if we wrap the styles outside hydration). I guess my question is where to put the CSS?